cypress/integration/customizedResources/user/add/AddRecord.spec.js

```diff
@@ -0,0 +1,30 @@
+import faker from 'faker';
+import { customized, leftNavbar } from '../../../../support/cssCommonSelectors';
+import { common } from '../../../../support/texts';
+import { getFormValues } from '../../../../support/helpersMethods';
+import comp from '../../../../support/components';
+  
+const { inputs, boardView } = customized;
+let email = 'aaaaaa' + faker.internet.email().toLowerCase();
+
+describe('[Customized resources/ User] Add record to the user', function() {
+  it('Should go to user and add record', function() {
+    cy.server()
+      .route('POST', '/admin/api/resources/User/actions/new').as('userCreated')
```

Comment:
OPT: probably can be extracted to some util method like `on(created('User'), 'userCreated')`